packages/client/src/v2-events/features/events/components/Action/utils.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import { Draft, UUID } from '@opencrvs/commons/client'
+
+export function getEventDrafts(
+  eventId: UUID,
+  localDraft: Draft,
+  drafts: Draft[]
+) {
+  return drafts
+    .filter((d) => d.eventId === eventId)
```

Comment:
filtering this was probably redundant anyway since drafts are already being filtered there on line 114
```ts
const drafts = getRemoteDrafts(event.id)
```